src/Packager.php

```diff
@@ -435,7 +599,54 @@ private function getScormManifestSchema(): array
      */
     private function deleteManifestAndDefinitionFiles()
     {
-        unlink($this->getSource().DIRECTORY_SEPARATOR.self::XML_MANIFEST_FILE_NAME);
-        delete_directory_if_exists($this->getSource().DIRECTORY_SEPARATOR.self::DIRECTORY_FOR_DEFINITION_FILES);
+        unlink($this->getSource() . DIRECTORY_SEPARATOR . self::XML_MANIFEST_FILE_NAME);
+        delete_directory_if_exists($this->getSource() . DIRECTORY_SEPARATOR . self::DIRECTORY_FOR_DEFINITION_FILES);
+    }
+
+    /**
+     * Returns Metadata schema.
+     *
+     * @return array
+     *
+     * @throws VersionIsNotSupportedException
+     */
+    private function getMetadataSchema(): array
+    {
+        /** @var AbstractMetadataSchema $metadataSchemaClass */
+        switch ($this->getVersion()) {
+            case ScormVersions::SCORM__1_2__VERSION:
+                return [];
```

Comment:
If you get same result from different cases, you should combine these conditions into one.

For example:
```
case 'foo1':
case 'foo2':
    return 'bar';
```